doc/manpages/qvm-service.rst

```diff
@@ -137,9 +137,32 @@ disable-dns-server
     Enabling this service will result in an empty /etc/resolv.conf.
     The functionality is implemented in /usr/lib/qubes/setup-ip.
 
+lightdm:
+    Default: enabled in GPU and VNC GuiVM
```

Comment:
Default value is rather about what happens if this service is not set (neither enabled nor disabled). So, in all those cases the default is "disabled"